packages/material-ui/src/TableFooter/TableFooter.js

```diff
@@ -46,8 +57,10 @@ TableFooter.propTypes = {
   component: PropTypes.elementType,
 };
 
-TableFooter.defaultProps = {
-  component: 'tfoot',
-};
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
We should hold of with this for now. We can adjust our resolver later but this is different behavior for components between prod and dev. There's no reason to do this in an alpha.

Reply:
You are right! I should rename the property, thank you.